lib/create-app.js

```diff
@@ -169,7 +169,7 @@ function initWebId (argv, app, ldp) {
       // without permission by including the credentials set by the Solid server.
       const origin = req.headers.origin
       const userId = req.session.userId
-      if (!argv.host.allowsSessionFor(userId, origin)) {
+      if (req.path !== '/logout' && req.path !== '/goodbye' && !argv.host.allowsSessionFor(userId, origin)) {
```

Comment:
Why this addition?

Reply:
The Logout didn't work on WebID-OIDC server, I added request to solid-auth-client

Reply:
Makes sense. Let's remove this change, though, the logout cookie behavior will need to be handled separately, elsewhere.

Reply:
I have fixed chmods, may be it was changed my Samba server, it was changed for .acl som another files.

Reply:
Removed handle for `/logout` and `/goodbye`

Reply:
This addresses the symptoms, but not the cause, which should be addressed by https://github.com/solid/node-solid-server/pull/793